modules/Microsoft.Network/connections/deploy.bicep

```
@@ -89,6 +89,9 @@ param virtualNetworkGateway2 object = {}
 @description('Optional. The remote peer. Used for connection type [ExpressRoute].')
 param peer object = {}
 
+@description('Optional. The Authorization Key to connect to an Express Route Circuit. Used for connection type [ExpressRoute].')
+param expressRouteCircuitAuthorizationKey string = ''
```

Please rename to keep alignment with API https://learn.microsoft.com/en-us/azure/templates/microsoft.network/connections?pivots=deployment-language-bicep

```suggestion
param authorizationKey string = ''
```